src/Security/Authenticator/GitHubAuthenticator.php

```diff
@@ -115,7 +116,9 @@ public function start(Request $request, AuthenticationException $authException =
 
     public function onAuthenticationFailure(Request $request, AuthenticationException $exception)
     {
-        $this->requestStack->getCurrentRequest()->getSession()->getFlashBag()->add('error', 'Your GitHub user is not in the allowed organization, or the membership is private.');
+        /** @var Session $session */
+        $session = $this->requestStack->getCurrentRequest()->getSession();
```

Comment:
* `$request` can be null
* this kind of change brings totally nothing to the project

Reply:
Unfortunately, I can't make `$request` nullable because the vendor parent method doesn't define it as nullable.
Regarding the typehint, I added it because PHPStan couldn't find `getFlashBag()` and was throwing an error. I tried to remove it and the error is still there.

Reply:
I mean, "$request **CAN** be null". That's a fact.
The code **must** handle such case